10up/classifai
# Feature/429 add attributation and assign post to image generation

### Description of the Change

- Add "Image generated by OpenAI's DALL·E" attributation as image caption when importing generated images.
- Add new `classifai_dalle_caption` filter to modify this caption via PHP.
- Assign the current post to the imported attachment.



Closes #429 

### How to test the Change
1. Edit a post and generate a image using Dall-E
2. Import the image
3. Go to the media library
4. Verify the newly imported image is attached to the post you just edited.
5. Verify the newly imported image has the "Image generated by OpenAI's DALL·E" attributation as a caption.

### Changelog Entry

> Added - DALL-E default attributation as image caption.
> Added - Assign generated image to post.

### Credits

Props @derweili 


### Checklist:
